ui/components/multichain/account-details/account-details.js

```diff
@@ -64,63 +70,84 @@ export const AccountDetails = ({ address }) => {
   );
 
   return (
-    <Modal isOpen onClose={onClose}>
-      <ModalOverlay />
-      <ModalContent>
-        <ModalHeader
-          onClose={onClose}
-          onBack={
-            attemptingExport
-              ? () => {
-                  dispatch(hideWarning());
-                  setAttemptingExport(false);
-                }
-              : null
-          }
-        >
-          {attemptingExport ? t('showPrivateKey') : avatar}
-        </ModalHeader>
-        {attemptingExport ? (
-          <>
-            <Box
-              display={Display.Flex}
-              alignItems={AlignItems.center}
-              flexDirection={FlexDirection.Column}
-            >
-              {avatar}
-              <Text
-                marginTop={2}
-                marginBottom={2}
-                variant={TextVariant.bodyLgMedium}
-                style={{ wordBreak: 'break-word' }}
+    <>
+      <Modal isOpen={!showHoldToReveal} onClose={onClose}>
```

Comment:
Do you mean because this Modal has an isOpen of `!showHoldToReveal` and the HoldToRevealModal has an isOpen of `showHoldToReveal`?  What would you suggest to clarify this?

Reply:
i think its better when the comment above.